cli/index.ts

```diff
@@ -61,7 +66,9 @@ async function main() {
     .option('--auth-param <name=value>', 'Additional variables to pass to the token endpoint', parseVariables, new Map())
     .action(capture('Import', ({ job }) => ({ job }), importCube))
 
-  return program.parseAsync(process.argv)
+  return trace.getTracer('cube-creator-cli').startActiveSpan('run', span => {
```

Comment:
I think the better semantic would be one tracer per package, and name it with the name of the package, e.g. `@cube-creator/cli`